src/lib/y2network/connection_config/qeth.rb

```diff
@@ -23,6 +23,23 @@ module Y2Network
   module ConnectionConfig
     # Configuration for qeth connections
     class Qeth < Base
+      # @return [String] read bus id
+      attr_accessor :read_channel
+      # @return [String] write bus id
+      attr_accessor :write_channel
+      # @return [String] data bus id
+      attr_accessor :data_channel
+      # @return [Boolean] whether layer2 is enabled or not
+      attr_accessor :layer2
```

Comment:
at first view i miss logical link address (LLADDR) ... is it somewhere else?

Reply:
Yes, it was added to base:

https://github.com/yast/yast-network/pull/909/files#diff-828c91bf00f25c07b26cb97ff7952eddR56